_posts/2017-02-21-new-services-on-discover.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+status: publish
+published: true
+title: 'New Services on Discover'
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+date: 2017-02-21 22:06:11
+categories:
+- Featured
+tags:
+- agrc
+- discover
+- google
+- naip
+- sgid
+
+---
+
+Recently AGRC made some changes to the services available from our cloud server Discover (discover.agrc.utah.gov).
+To see these new enhancements you may need to _refresh_ the connection you use to access Discover. ArcMap users can
+right-click your connection to Discover in ArcCatalog and select `Refresh`. You might also want to clear your local cache.
+ArcMap users can go to the service’s `Layer Properties -> Cache` tab and select `Clear Local Cache Now`. Be patient
+as this could take several minutes. Another option is to completely clear your ArcMap cache by going
+to `Customize -> ArcMap Options -> Display Cache -> Clear Cache`.  
+
+The service additions and changes are listed below. We also renamed most of the services. The renaming of the services should not have any effect on projects or applications that contained any of the services previous to the renaming. The renaming is on the layer's display title only a does not have any effect on the urls or request parameters for the service.
+
+ - **Utah** (licensed) - This service is now a mix of the most recent NAIP 1-meter and Google 6-inch imagery. The service
+ now utilizes scale dependent rendering to display the color balanced NAIP imagery down to cache level 16,
+ then renders the Google imagery at level 17 (starting at approx. 1:6,500) and down to level 20.
+
+ - **Google** (licensed)- This new service is the replacement for the old **utah** service and has the
+ most recent Google imagery for Utah.
+
+ - **Basemap-Hybrid** (licensed)- This new service was created for convenience and performance for a dynamic mash-up of vector features with a transparent background from the **Basemap-Overlay** and the **Utah** imagery service.
+
+More information about the services from Discover can be found [here]({{"/data/sgid-base-map-services-arcmap/" | prepend:site.baseurl}}).
```

Comment:
`here` links are kind of rookie. Could you update that to something like 

More information about the services from Discover can be found in the [Using SGID Base Map & Imagery Services in ArcMap](#)